xline-client/src/types/auth.rs

```diff
@@ -9,7 +9,10 @@ pub struct AuthenticateRequest {
 }
 
 impl AuthenticateRequest {
-    /// New `AuthenticateRequest`
+    /// Creates a new `AuthenticateRequest`.
+    ///
+    /// `name` is the name of the user to authenticate,
```

Comment:
These two comments are a bit verbose. Just directly change the parameter names to `user_name` and `user_password`. Some places have parameter explanations, while others do not.

Reply:
done